org.kde.krita.json

```diff
@@ -0,0 +1,311 @@
+{
+    "app-id": "org.kde.krita",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "krita",
+    "rename-icon": "calligrakrita",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home"
```

Comment:
Does this need all of home? The file picker should give it access to specific files via the portal. Is this for plugins etc.?

Reply:
At least here I get the error message "File /home/user/file.png does not exist." from krita, when I try to open a file per drag&drop or on the command line while running it without "--filesystem".

Reply:
Yeah, drag and drop doesn't work via a portal, sadly. On the command line you'd have to add an extra permissions grant as well. OK, fine for now.